setup.cfg

```diff
@@ -19,6 +19,7 @@ package_dir =
 install_requires =
     bluesky
     pyepics
+    blueapi
```

Comment:
should update this to github main, although that may be complicated by pinned dependency conflicts

Reply:
I think we fix this properly by doing https://github.com/DiamondLightSource/dodal/issues/145